.gitignore

```diff
@@ -0,0 +1 @@
+xcuserdata/
```

Comment:
Ignoring `xcuserdata` prevents user-specific files with things like breakpoints and UI state from being committed.